cmd/kubeadm/app/preflight/checks.go

```diff
@@ -998,6 +1006,9 @@ func addCommonChecks(execer utilsexec.Interface, cfg kubeadmapi.CommonConfigurat
 		if containerRuntime.IsDocker() {
 			isDocker = true
 			checks = append(checks, ServiceCheck{Service: "docker", CheckIfActive: true})
+			if runtime.GOOS == "linux" {
```

Comment:
Why both the runtime and buildtime conditionals for the check?

Reply:
sure, we can call this as it's a no-op for darwin and windows.